src/dotnet-svcutil/lib/src/Bootstrapper/SvcutilBootstrapper.cs

```diff
@@ -123,22 +123,17 @@ internal async Task GenerateProjectAsync(bool keepBootstrapperDir, ILogger logge
                 this.MSBuildProj = await MSBuildProj.DotNetNewAsync(projectFullPath, logger, cancellationToken).ConfigureAwait(false);
                 this.MSBuildProj.AddDependency(svcutilPkgRef, true);
 
-                // Comment out code below for reasons: 1. it never used for .net core later than V2.1 since when the approach is always use TF from the generated project.
-                // 2. with below code applied when target framework is netstandard2.0 client machine require netcoreapp2.0 (obsolete) for bootstrapper to work
-                // 3. keep it here for future reference in case when we need definite bootstrapper TF version
-
-                // NOTE: If post v2.0 NetStandard ships a different version from NetCore the table below needs to be updated!
-                //var targetFramework = frameworkInfo.FullName;
-                //if (isSupportedTFM && frameworkInfo.IsKnownDnx)
-                //{
-                //    if (frameworkInfo.Name == FrameworkInfo.Netstandard)
-                //    {
-                //        targetFramework = FrameworkInfo.Netcoreapp + TargetFrameworkHelper.NetStandardToNetCoreVersionMap[frameworkInfo.Version];
-                //    }
-                //    this.MSBuildProj.TargetFramework = targetFramework;
-                //}
+                var targetFramework = frameworkInfo.FullName;
+                if (isSupportedTFM && frameworkInfo.Name != FrameworkInfo.Netstandard && frameworkInfo.Version.CompareTo(new Version(6, 0)) >= 0)
```

Comment:
We will still support net standard.

Reply:
netstandard support remains both before and after the change. The bootstrapper project serves the purpose of type reuse and is required to be an executable target, hence it cannot be netstandard. With this update, the bootstrapper project now targets the default .NET Core version installed on the machine when the user project targets netstandard. Previously, the bootstrapper project inadvertently used the default .NET Core version installed on the machine regardless of the target of the user project, which was unintended.